### PR TITLE
Upgrade Nuget Dependency to 0.2 for SubscriptionsAdmin to support moveSubscriptions

### PR DESCRIPTION
**Pull request by caross**
## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [X] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by deathly809** [comment]
@bganapa I am not sure how to fix this since it is not cause by any changes,

```
  Failing : The term 'Failing' is not recognized as the name of a cmdlet, function, script file, or operable program. 
  Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
  At line:1 char:103
```

**Comment by cormacpayne** [comment]
@caross @deathly809 @bganapa 

For future reference, there's a message below that error:

```
Test failures occured, check the files in src/TestResults/FailingTests.
```

Looking at that folder in the build artifacts I found that there was an RDFE test that failed, so I restarted the build to confirm it was a flaky test.

**Comment by bganapa** [comment]
@deathly809 Yes the error does not look like related to our changes. just retrying the CI test job for now..

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Move-AzsSubscription.ps1 [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+.EXAMPLE
+	Move user subscriptions from a delegated provider to the Default Provider.
+	$resourceIds = Get-AzsUserSubscription -Filter "offerName eq 'o1'" | where {$_.DelegatedProviderSubscriptionId -eq "798568b7-c6f1-4bf7-bb8f-2c8bebc7c777"} | Select -ExpandProperty Id
+	Move-AzsSubscription -ResourceId $resourceIds
+#>
+function Move-AzsSubscription
+{
+	[CmdletBinding(DefaultParameterSetName='Subscriptions_MoveSubscriptions')]
+	param(    
+		[Parameter(Mandatory = $false)]
```
Comment (bganapa) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Move-AzsSubscription.ps1]: Is this not Mandatory?

Comment (caross) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Move-AzsSubscription.ps1]: Correct, null defines a removal of delegated provider offer.

**Comment by bganapa** [comment]
@cormacpayne This should be good to merge?

**Comment by bganapa** [comment]
ps-sign https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/922/console

**Comment by cormacpayne** [comment]
@bganapa @deathly809 @caross would you mind resolving the merge conflicts?

**Comment by caross** [comment]
@cormacpayne Merged against preview.

File: src/StackAdmin/Azs.Subscriptions.Admin/Help/Move-AzsSubscription.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+## DESCRIPTION
+Move subscriptions between delegated provider offers.
+This process will only perform a rebranding,
+the underlying offer, plans, quotas for the subscriptions will not be altered.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Move user subscriptions to a delegated provider offer.
```
Comment (cormacpayne) [inline src/StackAdmin/Azs.Subscriptions.Admin/Help/Move-AzsSubscription.md]: @caross this description should be below the code block it's in, and the PowerShell snippet below should be in this code block 😀 This comment applies to all examples added

File: src/StackAdmin/Azs.Subscriptions.Admin/Tests/src/Subscriptions.Tests.ps1 [hunk 3]
```diff
@@ -100,13 +100,20 @@ InModuleScope Azs.Subscriptions.Admin {
...
         it "CheckNameAvailability" -Skip:$('CheckNameAvailability' -in $global:SkippedTests) {
             $global:TestName = 'CheckNameAvailability'
 
             Test-AzsNameAvailability -Name $global:TestAvailability -ResourceType $global:ResourceType
         }
+
+        It "TestMoveSubscription" {
+            $global:TestName = 'MoveSubscription'
+            $resourceIds = Get-AzsUserSubscription -Filter "offerName eq 'o1'" | Select -ExpandProperty Id
+            Move-AzsSubscription -DestinationDelegatedProviderOffer $Null -ResourceId $resourceIds
```
Comment (cormacpayne) [inline src/StackAdmin/Azs.Subscriptions.Admin/Tests/src/Subscriptions.Tests.ps1]: @caross can we add test coverage for `Test-AzsMoveSubscription`?

**Pull request merged**